tests/components/influxdb/test_init.py

```diff
@@ -1226,6 +1226,13 @@ async def test_event_listener_attribute_name_conflict(
             influxdb.DEFAULT_API_VERSION,
             influxdb.exceptions.InfluxDBClientError("fail"),
         ),
+        (
+            influxdb.DEFAULT_API_VERSION,
+            BASE_V1_CONFIG,
+            _get_write_api_mock_v1,
+            influxdb.DEFAULT_API_VERSION,
+            influxdb.exceptions.InfluxDBServerError("fail"),
+        ),
```

Comment:
This one was a test I missed during https://github.com/home-assistant/core/pull/37181. It seems if the rest API for Influx V1 returns a 5xx error then the client returns an `InfluxDBServerError` rather then an `influxDBClientError`. This error was being handled in the write loop but not at startup. So now the code is properly capturing and handling this error at startup and in the main loop and this test matches that change.